src/components/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,34 @@
+import "./ErrorPage.scss";
+
+export interface ErrorPageProps {
+  errorMessage: string;
+  originURL: string;
+}
+
+function ErrorPage({errorMessage, originURL}: ErrorPageProps) {
+  return (
+    <>
+      <section className="error-page">
+        <span>{errorMessage}</span>
+        <button
+          id="home-button"
```

Comment:
I guess it's not important here, but you should only use the `id` if you are absolutely sure that there won't be any other element with the same id. 
`home-button` is a very generic name that may be likely used somewhere else. Since this is a fullscreen error page, it's ok for me. Nevertheless I want you to think about it.

I saw that you are just using it to select the element within your tests. You can also use `data-testid`  - which I guess but I'm not completely sure - will be removed in the build process. But you can use it to call `getByTestId()` from the `testing-library`. (see https://testing-library.com/docs/queries/bytestid/ for more information).

Reply:
I've even considered using data-testid but in the end I was too lazy. Serves me right ;)